joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/FileUtil.scala

```diff
@@ -68,6 +99,16 @@ object FileUtil {
     }
   }
 
+  def writeBytes(file: Path, content: Iterator[Byte], bufferSize: Int = 8192): Unit = {
```

Comment:
We do not have a single use case where we really need to work on `Iterator[Bytes]`. 
So why not make our live easier and just use `Iterable[Byte]` as parameter type?